firebase-functions/functions/src/operations/GameOperations.ts

```diff
@@ -8,30 +8,38 @@ import { Result } from '../models/result';
 import { Round } from '../models/round';
 import { Game } from '../models/game';
 import { collections } from '../constants/collections';
-import { Room } from '../models/room';
-import { GameType } from '../models/game-type';
 import DocumentData = firestore.DocumentData;
 import QueryDocumentSnapshot = firestore.QueryDocumentSnapshot;
+import { getRoom } from './roomOperations';
 
-export async function addRoundToGame(roomId: string, players: QueryDocumentSnapshot<DocumentData>[]): Promise<void> {
+export async function getCurrentGame(roomId: string, players: QueryDocumentSnapshot<DocumentData>[]): Promise<Game> {
   const round = {
-    playerChoices: getPlayerChoices(players),
+    playersChoices: getPlayersChoices(players),
     timeStamp: Date.now(),
   } as Round;
 
   const room = await getRoom(roomId);
   const games = room.games ?? [];
-  const currentGameIndex = games?.findIndex(game => !game.lastOneActive);
+  const isItANewGame = !games[games.length - 1]?.lastOneActive;
 
-  if (currentGameIndex > -1) {
-    games[currentGameIndex].rounds.push(round);
+  if (isItANewGame) {
+    games.push(createNewGame(round));
   } else {
-    games.push(createNewGame(round, room.typeOfGame));
+    games[games.length - 1].rounds.push(round);
   }
   await admin.firestore().collection(collections.rooms).doc(roomId).update({ games });
+  return games[games.length - 1];
 }
 
-function getPlayerChoices(players: QueryDocumentSnapshot<DocumentData>[]): PlayerChoice[] {
+export async function addLastOneActiveToGame(lastOneActive: string, roomId: string): Promise<void> {
```

Comment:
Not to be picky, but `set...` might be preferable? After all, you set a property. Add implies there is some list involved.